hvplot/__init__.py

```diff
@@ -176,7 +176,7 @@ def _get_doc_and_signature(
 
     parameters = []
     extra_kwargs = _hv.core.util.unique_iterator(
-        valid_opts + kind_opts + converter._axis_options + converter._op_options
+        valid_opts + kind_opts + converter._axis_config_options + converter._resample_options
```

Comment:
With `_axis_config_options` instead of the old `_axis_options`, less options are going to exposed in the signature, correct? How about `_size_layout_options`, `_grid_legend_options`, ...?